rasa/nlu/extractors/duckling_http_extractor.py

```diff
@@ -89,6 +92,10 @@ def _locale(self):
             self.component_config["locale"] = locale_fix
         return self.component_config.get("locale")
 
+
+    def _timeout(self):
```

Comment:
I don't think it's necessary to create a method for this, as it' only getting the value from the dict.

Reply:
Addressed this review comment.